OpenSim/Examples/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -173,8 +173,20 @@ void addDeviceConsoleReporterToModel(Model& model, Device& device,
 // the exercise. The project should execute without making any changes (you
 // should see the unassisted hopper hop slightly).
 //------------------------------------------------------------------------------
-int main()
-{
+int main(int argc, char* argv[]) {
+    //==========================================================================
+    // Command line argument parsing.
```

Comment:
Could you give examples of the different possible invocations of the resulting executable?